plugins/hls-tactics-plugin/src/Wingman/CaseSplit.hs

```diff
@@ -105,5 +106,5 @@ splitToDecl fixity name ams = do
 iterateSplit :: AgdaMatch -> [AgdaMatch]
 iterateSplit am =
   let iterated = iterate (agdaSplit =<<) $ pure am
-   in fmap wildify . head . drop 5 $ iterated
+   in fmap wildify . (\xs -> atNote "Wingman.CaseSplit.iterateSplit" xs 5) $ iterated
```

Comment:
I would just use !!

Reply:
Right, this one is a bit funny because we know the list is infinite. Hmm.